src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
Similarly, the message for stopping work should also indicate the type of machine. You could change it to 'Excavator has stopped working'.